libs/books/feature/src/lib/book-search/book-grid/book-grid-item/book-grid-item.component.html

```diff
@@ -20,7 +20,8 @@
         (click)="addBookToReadingList(book)"
         [disabled]="book.isAdded"
       >
-        Want to Read
+        <div *ngIf="isFinished; else notFinished">Finished</div>
+        <ng-template #notFinished>Want to Read</ng-template>
```

Comment:
This is also overkill, you could solve the same problem with simple interpolation -> `{{ isFinished? 'Finished' : 'Want to Read }}`